src/app/modules/item/pages/item-edit-content/item-edit-content.component.html

```diff
@@ -1,3 +1,3 @@
 <alg-section icon="fa fa-book" label="Description">
-  <alg-textarea [value]="value" (textChange)="updateForm($event)"></alg-textarea>
+  <alg-textarea [parentForm]="parentForm" inputName="description"></alg-textarea>
```

Comment:
There are still change to item-edit, you didn't really start a PR from master 😔
How am I supposed to review that?